flink-table/flink-table-runtime/src/test/java/org/apache/flink/table/runtime/operators/aggregate/window/SlicingWindowAggOperatorTest.java

```diff
@@ -963,6 +965,10 @@ private static long epochMills(ZoneId shiftTimeZone, String timestampStr) {
         return localDateTime.toInstant(zoneOffset).toEpochMilli();
     }
 
+    private static TimestampData wrapTs(long epochMillis) {
```

Comment:
How about import static  `TimestampData.fromEpochMillis` and use `fromEpochMillis` directly instead of add an extra `wrapTs` method?

Reply:
That's better. Will fix it.